Dockerfile

```diff
@@ -14,8 +14,8 @@ COPY pkg/    pkg/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOFLAGS=-mod=mod go build -a -o manager ./cmd/manager
 
-# Copy the controller-manager into a thin image
-FROM gcr.io/distroless/static:nonroot
+# Use distroless as minimal base image to package the manager binary
+FROM registry.access.redhat.com/ubi8/ubi-micro:8.6 
```

Comment:
latest?

Reply:
is it a good idea to use floating tags? I thought the best practice is to avoid using `latest` is it not?

Reply:
it should be `latest`

Reply:
fixed